contributing.md

```diff
@@ -0,0 +1,81 @@
+contribute to projects that they use frequenty. Whether adding a feature
+to a project, fixing a bug, or adding documentation, it allows communities
+to form out of these projects.
+
+## Where to find projects
+
+Many widely known projects such as Firefox and VLC are open sourced and available 
+to be contributed by members of the community. Sites such as Github or Bitbucket
+allow software to be hosted on Repositories and shared with the world.
+
+If there is a project that one uses frequently, it's possible to contribute
+to such project, although it's recommended to view if the Repository has
+a contributing file or license so one knows what they are allowed to do.
+
+If the project allows outside contributions, one is ready to help out.
+
+## How to contribute
+
+There are multiple ways of improving Open Source Software. A few of the main
+ways include writing of fixing documentation, posting bugs or feature requests,
+and even fixing part of the code itself. On Github, there is a button in the
+top corner of every repository that says Fork. By default a normal user can't
+edit code in repositories other than their own or ones they have permission to
+edit. Forking a repository allows the creation of personal copies of the
+repository on your own Github account so you can make changes to the codebase.
+
+## Issues
+
+If someone has a problem with a piece of Open Source Software, they can add
+issues to the project's Github Repo or a specified bug tracker, depending on
+the size of the project. The issues can include a bug discovered in the code,
+a feature that seems relevant to the codebase that would be useful if added,
+or a feature that may be confusing to understand and could be restructured
+to be easier to use.
+
+Users are allowed to either post an issue, or view and help fix an issue in
+the repo. These issues are threads that allow discussions on ways to fix
+bugs in software or make a better understanding on how the project works.
+
+## Documentation
+
+A resource that allows users of a piece of software to understand how to use
+it is the documentation. documentation consists of an overview of what the 
+sofware is, how to install and use the it, common issues and troubleshooting,
+and ways of contributing to the project. These documents are not final products
+and are often updated to include more information to contributors or users of
+the project or to make the current documentation easier to understand.
+contributors are often able to contribute to this documentation and help out
+the maintainers of the project, and the process is explained in the next section.
+
+## Branching and Pull Requests
+
+A convention that some people use when working on projects is to have a
+single branch of the codebase that is stable and different branches for
+adding new features to the repository. By default, a repository has a
+master branch when created, while other branches need to be created
+seperately.
```

Comment:
Also you should specify somewhere that this document is very git-centric, and ignores non-distributed version control systems entirely.